code/datums/uplink_item.dm

```diff
@@ -511,3 +549,35 @@ var/list/uplink_items = list()
 		U.uses -= max(0, I.cost)
 		feedback_add_details("traitor_uplink_items_bought","RN")
 		return new I.item(loc)
+
+/datum/uplink_item/badass/surplus_crate
+	name = "Syndicate Surplus Crate"
+	desc = "A crate containing 50 telecrystals worth of random syndicate leftovers."
+	cost = 20
+	item = /obj/item/weapon/storage/box/syndicate
+	excludefrom = list(/datum/game_mode/nuclear)
```

Comment:
If you can't get the crate during nuclear games, when else will you have 20 telecrystals?  I am confused